packages/react-native-reanimated/src/layoutReanimation/web/animation/Zoom.web.ts

```diff
@@ -105,14 +108,16 @@ export const ZoomOutData = {
   ZoomOutLeft: {
     name: 'ZoomOutLeft',
     style: {
-      100: { transform: [{ translateX: '-100vw', scale: 1 }] },
+      0: { transform: [{ translateX: '0vw', scale: 1 }] },
+      100: { transform: [{ translateX: '-100vw', scale: 0 }] },
```

Comment:
Here I changed `scale` to `0` - probably we have overlooked that during first review of LA 😅